server/src/api/resources/trip/trip.restRouter.js

```diff
@@ -7,11 +7,11 @@ tripRouter.param("id", tripController.findByParam)
 
 tripRouter
   .route("/")
-  .get(tripController.getAll)
-  .post(tripController.createOne)
+  .get(tripController.getAllTrips)
+  .post(tripController.createTrip)
 
 tripRouter
   .route("/:id")
-  .get(tripController.getOne)
-  .put(tripController.updateOne)
-  .delete(tripController.deleteOne)
+  .get(tripController.getOneTrip)
+  .put(tripController.updateTrip)
+  .delete(tripController.deleteTrip)
```

Comment:
Might be a good idea to add:
```javascript
tripRouter
  .route("/:id/waypoints")
  .get(tripController.populateWaypoints)
```
Be handy to have a endpoint to retrieve only just the waypoints associated with the Trip.

Reply:
got it 👍